prediction/src/algorithms/segment/src/levelset_segment.py

```diff
@@ -0,0 +1,37 @@
+import SimpleITK as sitk
+
+
+def segment_nodule(image: sitk.Image, nodule: dict):
+    """
+
+    :param image: Input image
+    :param nodule: dict containing the nodule coordinates
+    :return:
+    """
+
+    seed = (nodule["x"], nodule["y"], nodule["z"])
+    seg = sitk.Image(image.GetSize(), sitk.sitkUInt8)
+    seg.CopyInformation(image)
+    seg[seed] = 1
+    seg = sitk.BinaryDilate(seg, 3)
+
+    init_ls = sitk.SignedMaurerDistanceMap(seg, insideIsPositive=True, useImageSpacing=True)
+    lsFilter = sitk.ThresholdSegmentationLevelSetImageFilter()
+    lsFilter.SetLowerThreshold(-600)
```

Comment:
Could you explain these magic numbers in meaningful constant variables, please?

Reply:
Mmm, please do so :)